src/PrestaShopBundle/Form/Validator/Constraints/TinyMceMaxLengthValidator.php

```diff
@@ -40,8 +44,9 @@ public function validate($value, Constraint $constraint)
             "\n\r",
         ];
         $str = str_replace($replaceArray, [''], strip_tags($value));
+        $length = iconv_strlen($str);
```

Comment:
@jolelievre just curious, why not `mb_strlen`? is `iconv_strlen` works correctly if we decide to use emojis or some special characters?

Reply:
Could you add Unit tests, please ?

Reply:
@kpodemski I have no idea why `iconv_strlen` was used ^^ It was used before I touched this file and I didn't change it 🤷‍♂️

@Progi1984 this validator already existed for a long time and had no unit tests, so this PR's scope shouldn't be to add them (you didn't add some when you modified it for PHPStan right?) But I'm not gonna lose time arguing and will add them to make you happy regardless ^^

Reply:
I'm sorry @jolelievre. You add new behavior in code. It's why we add tests.

Reply:
I didn't add any new behaviour but anyway..